ath79: add tl-wr1043nd-v4 support

lucize opened the pull request:
tplink-safeloader: resize kernel partition
kernel 4.14 is bigger and it will not fit anymore, resize the partition, we have plenty of space left

everything is working

Signed-off-by: Lucian Cristian <lucian.cristian@gmail.com>

lucize commented:
based on #1043

hackpascal commented:
Hi, please be noticed that I've re-pushed the commit and the file qca956x.dtsi has been changed.

lucize commented:
I'll drop the file so it will have no conflicts after yours PR gets merged, the dtsi is also working for ubiquiti ap ac lite/lr

karlp commented on target/linux/ath79/dts/qca9563_tl-wr1043n.dtsi:
```diff
@@ -0,0 +1,223 @@
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+				partition@0 {
```
indentation is wrong here.

karlp commented on target/linux/ath79/dts/qca9563_tl-wr1043n.dtsi:
```diff
@@ -0,0 +1,223 @@
+				read-only;
+			};
+
+			partition@20000 {
```
just fyi, you can also just do "firmware@20000" and skip the label properties.

lucize committed:
ath79: add tl-wr1043nd-v4 support
everything is working

tplink-safeloader: resize kernel partition
kernel 4.14 is much bigger, resize the partition

Signed-off-by: Lucian Cristian <lucian.cristian@gmail.com>

blogic commented:
merged, Thanks !
